### ISSUE TITLE
Issue when not providing --work-dir

### ISSUE THREAD

**Issue by marcodelapierre** [seqeralabs/tower-agent#45]
Tower Agent version 0.4.3 (build 87f4d1f)

Issue:
```
$ tw-agent $TOWER_CONNECTION_ID
13:23:05.272 INFO - Established active environments: [cli]
13:23:05.406 ERROR- The work directory '?/work' do not exists. Create it or provide a different one using '--work-dir'.
```

It looks like the Tower-agent is looking for a weird work dir name, `?/work`.

From the help screen, it should be  `~/work`.

**Comment by marcodelapierre** [comment]
This seems to be the source line in question:

https://github.com/seqeralabs/tower-agent/blob/master/src/main/java/io/seqera/tower/agent/Agent.java#L227

For some reason, `System.getProperty("user.home")` does not return what it should return.

**Comment by pditommaso** [comment]
We are seen crazy errors with Java native compilation and env variables. I'll leave @jordeu for a detailed response

**Comment by jordeu** [comment]
Yes, this happens only in some systems and is some how related to GraalVM native compilation. I think that the ongoing changes that we track at this [issue](https://github.com/seqeralabs/tower-agent/issues/37) will fix also this. I'll keep this open until we finish that GraalVM update.

**Comment by jordeu** [comment]
Also in latest releases we are also providing a `tw-agent.jar` file that you can run directly with a JDK v11 or above like this:
```
java -jar tw-agent.jar <connection_id> ...
```
With the pure Java version you won't have this problem.
